react/DynamicIframe.tsx

```diff
@@ -0,0 +1,74 @@
+import Iframe from './Iframe'
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+const DynamicIframe: StorefrontFunctionComponent<DynamicIframeProps> = ({
+  srcPrepend,
+  dynamicParams,
+  srcAppend,
+  width,
+  height,
+  title,
+}) => {
+  const {
+    route: { params },
+  } = useRuntime()
+  let dynamicSrcValues = ''
+  if (dynamicParams && dynamicParams.length) {
+    dynamicParams.forEach(thisValue => {
+      if (params[thisValue]) {
+        dynamicSrcValues += '/' + params[thisValue]
+      }
+    })
+  }
+  const newSrc =
+    srcPrepend +
+    dynamicSrcValues +
+    (srcAppend && srcAppend != '' ? '/' + srcAppend : '')
+  return <Iframe title={title} src={newSrc} width={width} height={height} />
```

Comment:
Why don't we just map the dynamic params to the right place?

Example:

`http://{account}.foobar.com/{param2}/{param1}`

Reply:
Define right place? 
Marc's code allows the user to use some or all of the available params, in whatever order they want. I think that's more robust ... your thoughts are welcome.

Reply:
it's also designed to allow the individual storefronts to specify how many params they want to use (some sites might need/want more than 2 params) and what their params are called so that they can use variable names that make sense for their use.

Reply:
What I'm trying to say is the position of the params should not be fixed.

Given the src `http://{param1}.foobar.com/{param2}/static/{param1}.aspx`

It could be possible to place param1 and param2 wherever you want in it.

Right now it's only possible to do this:

`${srcPreprend}/${params}/${srcAppend}`

Reply:
(I'm using param1 and param2 just as examples here)

Reply:
thanks for the feedback, I've updated the PR to hopefully accomplish this